userspace/falco/app_actions/print_ignored_events.cpp

```diff
@@ -28,7 +28,7 @@ void application::print_all_ignored_events()
 	std::set<string> ignored_event_names;
 	for(uint32_t j = 0; j < PPM_EVENT_MAX; j++)
 	{
-		if(!sinsp::simple_consumer_consider_evtnum(j))
+		if(!simple_consumer_consider(etable[j].flags))
```

Comment:
In this function we print together event names and an syscall names :thinking:

Reply:
Yep...i am not sure how useful is this function, but it was there and we are not going to break user cmdline API :)